src/api/status/src/server.js

```diff
@@ -43,5 +44,15 @@ if (process.env.PATH_PREFIX) {
   service.router.use('/', serveStatic(path.join(__dirname, '../public')));
 }
 
+service.router.get('/v1/status/status', (req, res) => {
```

Comment:
`/v1/status` instead of `/v1/status/status`

Reply:
@HyperTHD the endpoint defined to get the status is `/status`, which makes the whole route `/v1/status/status`

Reply:
Yup I think API wouldn't work if we changed to `/v1/status` since `/v1/status` fetch the static site